suricata/update/commands/listsources.py

```diff
@@ -38,6 +40,9 @@ def list_sources():
             logger.warning("%s: will use bundled index.", err)
     index = sources.load_source_index(config)
     for name, source in index.get_sources().items():
+        is_free = source.get("subscribe-url")
```

Comment:
This should be the reverse. What is happening now is that ```--free``` lists all the non-free sources.

Reply:
OK. Thanks.

Reply:
Fixed in #54